.chloggen/jaegerremotesampling_replace_thrift-gen_with_proto-gen.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: jaegerremotesampling
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Replace Thrift-gen with Proto-gen types for sampling strategies
+
+# One or more tracking issues related to the change
+issues: [18401]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
Can you explain the details of the change being `breaking` in the subtext? How are current clients impacted? 
Most importantly (to me, selfishly), what does this mean for _OTEL_ JaegerRemoteSampler implementations (more so than for Jaeger agents)?

Reply:
(Hopefully the breakage is limited to JSON client interactions and not at all impacting binary encoded proto requests i.e., gRPC?)